app/converters/sync_break_down_structures_converter.rb

```diff
@@ -0,0 +1,28 @@
+module SyncBreakDownStructuresConverter
+  include BodConverter
+
+  def process_sync_noun(noun)
+    begin
+      floc = FunctionalLocation.find_by uuid: noun.dig("breakdownStructure","networkForSegment","segment","uUID")
+      unless floc.present?
+        floc = create_missing_floc(noun.dig("breakdownStructure","networkForSegment","segment"), "SyncBreakDownStructures")
+      end
+
+      noun.dig("breakdownStructure","connection").each do |connection|
+
+        bds = BreakDownStructure.find_by(uuid: noun.dig("breakdownStructure","uUID"), from_uuid: connection.dig("from","uUID"), to_uuid: connection.dig("to","uUID"))
```

Comment:
```suggestion
        # TODO: Should probably preserve the connection's UUID as well, in case we do updates to breakdown structures in future.
        bds = BreakDownStructure.find_by(uuid: noun.dig("breakdownStructure","uUID"), from_uuid: connection.dig("from","uUID"), to_uuid: connection.dig("to","uUID"))
```